modules/installation-user-defined-tags-azure.adoc

```diff
@@ -3,78 +3,16 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="installing-azure-user-defined-tags_{context}"]
-= Configuring user-defined tags for Azure
+= Configuring user-defined tags for {azure-short}
 
-In {product-title}, you can use the tags for grouping resources and for managing resource access and cost. You can define the tags on the Azure resources in the `install-config.yaml` file only during {product-title} cluster creation. You cannot modify the user-defined tags after cluster creation.
+In {product-title}, you can use tags for grouping resources and for managing resource access and cost. Tags are applied only to the resources created by the {product-title} installation program and its core operators such as Machine API Operator, Cluster Ingress Operator, Cluster Image Registry Operator. The {product-title} consists of the following types of tags:
 
-Support for user-defined tags is available only for the resources created in the Azure Public Cloud. User-defined tags are not supported for the {product-title} clusters upgraded to {product-title} {product-version}.
+* {product-title} tags: By default, {product-title} installer attaches the {product-title} tags to the {azure-short} resources. These {product-title} tags are not accessible for the users. The format of the {product-title} tags is `kubernetes.io_cluster.<cluster_id>:owned` where `cluster_id` parameter is the value of `.status.infrastructureName` in `config.openshift.io/Infrastructure` API.
 
-User-defined and {product-title} specific tags are applied only to the resources created by the {product-title} installer and its core operators such as Machine api provider azure Operator, Cluster Ingress Operator, Cluster Image Registry Operator.
+* User-defined tags: User-defined tags are manually created in `install-config.yaml` file during installation. When creating the user-defined tags, you must consider the following points:
 
-By default, {product-title} installer attaches the {product-title} tags to the Azure resources. These {product-title} tags are not accessible for the users.
+** User-defined tags on {azure-short} resources can only be defined during {product-title} cluster creation, and cannot be modified after the cluster is created.
 
-You can use the `.platform.azure.userTags` field in the `install-config.yaml` file to define the list of user-defined tags as shown in the following `install-config.yaml` file.
+** Support for user-defined tags is available only for the resources created in the {azure-short} Public Cloud.
```

Comment:
Consider using a [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/) for items like this, i.e.:

```
{product-title} tags:: By default, {product-title} installer attaches the {product-title} tags to the {azure-short} resources. These {product-title} tags are not accessible for the users. 
+
The format of the {product-title} tags is `kubernetes.io_cluster.<cluster_id>:owned` where `cluster_id` parameter is the value of `.status.infrastructureName` in `config.openshift.io/Infrastructure` API.

User-defined tags:: User-defined tags are manually created in `install-config.yaml` file during installation. When creating the user-defined tags, you must consider the following points:
+
* User-defined tags on {azure-short} resources can only be defined during {product-title} cluster creation, and cannot be modified after the cluster is created.
* Support for user-defined tags is available only for the resources created in the {azure-short} Public Cloud.
* User-defined tags are not supported for the {product-title} clusters upgraded to {product-title} {product-version}.
```

That looks something like this (VSC preview doesn't render the attributes, but so you can see the format):
![image](https://github.com/user-attachments/assets/9984adbc-a914-4962-a1dc-3eea7b8bc361)

Reply:
sure, thanks!